test/cpp/thread_manager/thread_manager_test.cc

```diff
@@ -91,7 +91,7 @@ grpc::ThreadManager::WorkStatus ThreadManagerTest::PollForWork(void **tag,
   // Simulate "polling for work" by sleeping for sometime
   SleepForMs(kPollingTimeoutMsec);
 
-  *tag = nullptr;
+  *tag = NULL;
```

Comment:
I got the following error on gcc4.4 portability tests: 
```
test/cpp/thread_manager/thread_manager_test.cc: In member function 'virtual grpc::ThreadManager::WorkStatus ThreadManagerTest::PollForWork(void**, bool*)':
test/cpp/thread_manager/thread_manager_test.cc:94: error: 'nullptr' was not declared in this scope
```
(full details https://grpc-testing.appspot.com/job/gRPC_portability_pull_requests_linux/575/testReport/junit/(root)/aggregate_tests/run_tests_c___linux_dbg_x64_gcc4_4/ )

I initially thought adding `grpc++/impl/codegen/config.h` would help because that seems to define nullptr. That still didn't help :(